client/src/components/lobby/duels-skeleton.tsx

```
@@ -7,32 +9,69 @@ export function DuelsSkeleton() {
-          <tr
-            key={index}
-            className={clsx(
-              !isLast && "border-b border-[#23272F]",
-              "w-[814px] bg-[#0F1116] flex flex-row items-center",
-            )}
-          >
-            <td className="flex flex-row items-center p-4 space-x-5 w-[200px] justify-start">
-              <div className="flex flex-row items-center space-x-5 w-fit">
-                <div className="w-8 h-8 bg-gray-700 rounded-full"></div>
-                <div className="w-24 h-4 bg-gray-700 rounded"></div>
+          <div className="w-[874px] h-[874px] bg-[url('./assets/lobby-box-long.png')] bg-contain bg-no-repeat p-8">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)